### PR TITLE
Edit links to the Planning guide

### PR DESCRIPTION
**Pull request by spetrosi**
Issue #192

### CODE REVIEW

File: guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc [hunk 1]
```diff
@@ -308,7 +308,7 @@ This is the only method with which you can provision hosts in IPv6 network.
 .Prerequisites
 
 * Ensure that you meet the requirements for HTTP booting.
-For more information, see {BaseURL}/planning_for_red_hat_satellite/chap-red_hat_satellite-architecture_guide-provisioning_concepts#http_booting_requirements[HTTP Booting Requirements] in _Planning for {Project}_.
+For more information, see {PlanningDocURL}http-booting-requirements[HTTP Booting Requirements] in _Planning for {Project}_.
```
Comment (spetrosi) [inline guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc:311]: I added a new ID for this section in `guides/doc-Planning_Guide/topics/Provisioning_Concepts.adoc`, this link will work after this PR is merged.

**Review by melcorr — APPROVED** [review]
Works for me!

**Comment by melcorr** [comment]
Pls rebase

**Comment by spetrosi** [comment]
Rebased @melcorr, thanks for the review.

**Pull request merged**